src/components/TeaserFront/Lead.js

```diff
@@ -8,26 +8,29 @@ import {
   serifRegular19,
   serifRegular23
 } from '../Typography/styles'
+import { convertStyleToRem, pxToRem } from '../Typography/utils'
 
 const leadStyle = {
-  ...serifRegular19,
-  lineHeight: '27px',
+  ...convertStyleToRem(serifRegular19),
+  lineHeight: pxToRem('27px'),
   margin: '0 0 10px 0',
-  color: colors.text
+  color: colors.text,
+  wordBreak: 'break-word',
```

Comment:
This is used in many placed and could lead to unwanted breaks. I'd prefer to not add it. This was for «Lese sie auch» right?